### PR TITLE
Testing new branch CLA stuff

### PR DESCRIPTION
**Pull request by dangelyjeanelle**
This is a test

### CODE REVIEW

File: src/main/java/com/google/sps/data/Member.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        return new Member(alias);
+    }
+    public String getAlias(){
+        return this.alias;
+    }
+    public static EmbeddedEntity toEmbeddedEntity(Member member) {
+        EmbeddedEntity memberEntity = new EmbeddedEntity();
+        memberEntity.setProperty(ALIAS_PROPERTY, member.getAlias());
+        return memberEntity;
+    }
```
Comment (jcderry5) [inline src/main/java/com/google/sps/data/Member.java:22]: nit: new line after each method

File: src/main/java/com/google/sps/data/Message.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+    //Returns the body of the message
+    public String getMessageBody(){
+        return this.message;
+    }
+    //Returns when the message was sent
+    public Long getTimestamp(){
+        return this.timestamp;
+    }
+
```
Comment (jcderry5) [inline src/main/java/com/google/sps/data/Message.java:40]: nit: remove new line

File: src/main/java/com/google/sps/data/Room.java [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    //Get all of the members as a list of EmbeddedEntities
+    private List<EmbeddedEntity> getMembersAsEmbeddedEntities(){
+        return members.stream().map(Member::toEmbeddedEntity).collect(Collectors.toList());
+    }
+
+    //Returns the Room's video url list
+    public Queue<Video> getVideos() {
+        return this.videos;
+    }
+
```
Comment (jcderry5) [inline src/main/java/com/google/sps/data/Room.java:70]: nit: remove new lines after methods

File: src/main/java/com/google/sps/servlets/JoinRoomServlet.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  private String inputtedUserTag = "user-party-link";
+  private String currentRoomId = "";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    /** TODO: Create hard-coded hashmap of {Room ID : URL} Hashmap
+        TODO: Print json-ified string to /join-room page for private room to fetch */
+  }
+
+	@Override
```
Comment (jcderry5) [inline src/main/java/com/google/sps/servlets/JoinRoomServlet.java:39]: nit: remove indentation

File: src/main/webapp/scripts/private-room-script.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+	event.target.playVideo();
+}
+
+// 5. The API calls this function when the player's state changes.
+//    The function indicates that when playing a video (state=1),
+//    the player should play for six seconds and then stop.
+var done = false;
+function onPlayerStateChange(event) {
+	if (event.data == YT.PlayerState.PLAYING && !done) {
+		setTimeout(stopVideo, 6000);
```
Comment (jcderry5) [inline src/main/webapp/scripts/private-room-script.js:56]: nit: fix indentation

**Comment by dangelyjeanelle** [comment]
@googlebot I fixed it.

**Comment by dangelyjeanelle** [comment]
@googlebot I fixed it.

**Pull request merged**